code/datums/syndicate_buylist.dm

```diff
@@ -1052,7 +1052,7 @@ ABSTRACT_TYPE(/datum/syndicate_buylist/traitor)
 
 /datum/syndicate_buylist/traitor/rose
 	name = "Poison rose"
-	items = list(/obj/item/plant/flower/rose/poisoned)
+	items = list(item = /obj/item/clothing/head/flower/rose/poisoned)
```

Comment:
```suggestion
	items = list(/obj/item/clothing/head/flower/rose/poisoned)
```
I don't think you meant to do this

Reply:
Oops totally missed that, fixed now thanks